### PR TITLE
so/burger menu  version 2.2

### LINKED ISSUE #54: Burger menu: Version 2.0
**Nb** Menu items as follows: 

- About
- Manifesto
- Contact 
- Advertising
- Buy

Screenshot 2020-07-13 at 15 28 58

iPhone 8 Plus - 3

### PR DESCRIPTION
**Pull request by sssoja**
### What changes have you made?

- added `GlobalStyles.tsx` component 
- fixed display issue on overlay styles
- developed out responsive styles

### Which issue(s) does this PR fix?
Fixes #54

### Screenshots (if there are design changes)
Screenshot 2020-07-23 at 21 31 05
Screenshot 2020-07-23 at 21 31 15

### How to test
`yarn start` and hit the menu button

### CODE REVIEW

File: src/components/Home.tsx [hunk 1]
```diff
@@ -1,100 +1,88 @@
...
 import NavMenu from "./NavMenu";
+import oxymore from "./assets/home-page/oxymore.png";
+import manifesto from "./assets/home-page/manifesto.png";
+import number from "./assets/home-page/number-one.png";
+import alpha from "./assets/home-page/360_alpha.png";
 
-// background
-type BgProps = BackgroundProps & ColorProps & LayoutProps & FlexboxProps;
+const Main = styled.main<SpaceProps & FlexboxProps & GridProps>`
+  background: black;
```
Comment (mantagen) [inline src/components/Home.tsx:22]: `colors.black`?

File: src/components/NavMenu.tsx [hunk 2]
```diff
@@ -2,138 +2,160 @@ import React, { useState, Fragment } from "react";
...
-  &:link,
-  &:active {
-    text-decoration: none;
+`;
+
+const MenuText = styled.li`
+  transition: transform 0.4s;
+  &:hover {
+    transform: scale(1.01);
+    transform-origin: left;
```
Comment (mantagen) [inline src/components/NavMenu.tsx:73]: This line needs to be outside of the hover, otherwise immediately after the hover the transform jumps to the left!

Comment (sssoja) [reply src/components/NavMenu.tsx]: yeeeeh

**Comment by sssoja** [comment]
Closing to open new branch because of merge conflicts

**Pull request closed without merging**